lib/Graph/Graph.cpp

```diff
@@ -581,7 +581,7 @@ Node *Function::createBroadcast(llvm::StringRef name, NodeValue input,
   // new shape (no action taken) or == 1 (broadcast in that direction). Else
   // the original shape had no dimensions here (after considering axis), so
   // add the new dimension and broadcast in that direction.
-  size_t reshapeDims[newShape.size()];
```

Comment:
GLOW_VLA is not very readable. I prefer to just use array of size "max_tensor_dimensions" here.

Reply:
VLA means Variable Length Array， msvc doesn't support for that， so I place it in support.h， the name comes form 
https://github.com/jerryscript-project/jerryscript/blob/a74bf7dc91e7358f0e6c7e111e727110377fca82/jerry-core/include/jerryscript-compiler.h#L62,
Also are my patch, I hope this would have an agreement

Reply:
I know what VLA is. I am saying that GLOW_VLA is not very readable. I prefer to use the static size: int foo[max_tensor_dimensions];

Reply:
so does max_tensor_dimensions a constants or a macro？

Reply:
It does not matter if it’s a macro or a constant number. It is not dynamic.

Reply:
I'll try constant number first, if doesn't works, macro would used.